src/registrar/templates/django/admin/domain_change_form.html

```diff
@@ -8,8 +8,12 @@
 
 {% block field_sets %}
     <div class="submit-row">
+        {% if original.state == original.State.READY %}
```

Comment:
Since you mentioned it during the demo: I think this approach is fine here. I don't have any objections to this

Reply:
thanks for looking at this